dolfyn/tools/misc.py

```diff
@@ -204,7 +161,7 @@ def slice1d_along_axis(arr_shape, axis=0):
         k += 1
 
 
```

Comment:
I think we want to expose this function to the API. It's a widely used tool in ADCP analysis.

Reply:
I kept it because it is so widely used in the code, but unless I'm mistaken it's easily replaceable by xarray's selection capabilities. Take ADCP data in beam coordinates - if you want a slice along any single data, you have `dat.sel(dir=4, range=5)` returns a slice of all data from beam 4 in the 5m depth bin, or any dimension or combination thereof you desire.